src/librustc_mir/borrow_check/diagnostics/mod.rs

```diff
@@ -537,20 +537,37 @@ pub(super) enum UseSpans {
         // The span of the first use of the captured variable inside the closure.
         var_span: Span,
     },
+    // The access is caused by using a variable as the receiver of a method
+    // that takes 'self'
+    FnSelfUse {
+        // The span of the variable being moved
+        var_span: Span,
+        // The span of the method call on the variable
+        fn_call_span: Span,
```

Comment:
Could we change this to cover only from the ident of the method (after the dot) until the closing paren?

```
foo.bar();
    ^^^^^
```

Reply:
That kind of span isn't currently available - it will be added by https://github.com/rust-lang/rust/pull/73182

Reply:
Now that #73182 has landed, I've updated the code to use that span